src/main/java/net/dv8tion/jda/internal/entities/sticker/RichStickerImpl.java

```diff
@@ -90,28 +88,6 @@ public String setDescription(String description)
     @Override
     public String toString()
     {
-        return "RichSticker:" + getType() + ':' + name + '(' + getId() + ')';
-    }
-
-    @Override
-    public int hashCode()
-    {
-        return Objects.hash(id, format, name, getType(), tags, description);
-    }
-
-    @Override
-    public boolean equals(Object obj)
-    {
-        if (obj == this)
-            return true;
-        if (!(obj instanceof RichStickerImpl))
-            return false;
-        RichStickerImpl other = (RichStickerImpl) obj;
-        return id == other.id
-                && format == other.format
-                && getType() == other.getType()
-                && Objects.equals(name, other.name)
-                && Objects.equals(description, other.description)
-                && Helpers.deepEqualsUnordered(tags, other.tags);
+        return "RichSticker[" + getType() + "]:" + name + '(' + getId() + ')';
```

Comment:
Is it actually possible to get this considering the implementers override it?

Reply:
If future sticker types are added and we forget to override toString, this could be used. I think its fine to just keep it.